samtranslator/model/stepfunctions/events.py

```diff
@@ -173,6 +179,7 @@ class CloudWatchEvent(EventSource):
         "DeadLetterConfig": PropertyType(False, is_type(dict)),
         "RetryPolicy": PropertyType(False, is_type(dict)),
         "State": PropertyType(False, is_str()),
+        "Target": PropertyType(False, is_type(dict)),
```

Comment:
Should be `Property` or `PassThroughProperty`, see https://github.com/aws/serverless-application-model/blob/develop/DEVELOPMENT_GUIDE.md#development-guidelines

Reply:
Oh yes! Will update it